Makefile

```diff
@@ -24,23 +23,21 @@ deps:
 envars-webhook: $(shell find . -name '*.go')
 	TMPDIR=/var/tmp CGO_ENABLED=0 GOOS=linux go build -ldflags="-s -w" -o $@ ./cmd/envars-webhook
 
-image: envars-webhook deps
```

Comment:
deps was added here to ensure module dependencies are resolved in a new environment (i.e. such as a build pipeline)